package.json

```diff
@@ -2,25 +2,23 @@
   "name": "gavel",
   "version": "0.0.0-semantically-released",
   "description": "Validator of HTTP transactions (JavaScript implementation)",
-  "main": "lib/gavel.js",
+  "main": "src/gavel.js",
   "engines": {
     "node": ">= 4"
   },
   "bin": {
     "gavel": "bin/gavel"
   },
   "scripts": {
-    "build": "coffee -b -c -o lib/ src/",
     "lint": "coffeelint src",
     "test": "npm run test:server && npm run test:browser && npm run test:features",
     "test:server": "mocha \"test/unit/**/*-test.coffee\"",
     "test:browser": "mochify \"test/unit/**/*.coffee\" --transform=coffeeify --extension=.coffee",
     "test:features": "coffee scripts/cucumber.coffee",
-    "prepublish": "npm run build",
     "coverage": "scripts/cov",
     "coveralls": "npm run coverage mocha-lcov-reporter | coveralls",
     "ci:lint": "npm run lint",
-    "ci:test": "npm run test && npm run coveralls",
```

Comment:
Temporary removed. Will be introduced back as a part of the next pull request.

Reply:
Mentioned here: https://github.com/apiaryio/gavel.js/issues/96#issuecomment-484012311

It's a part of decaffeination, so it's under the respective issue. As I've mentioned, this branch is not going to `master`, it's but a step in this venture :)